website/static/js/jquery.homeycombs.js

```diff
@@ -0,0 +1,145 @@
+(function($) {

```

Comment:
I think having most things in our server will not hurt. Loading files from different soureces means multiple domain lookups and fetching.